integrationTests/frontend/wallet/txInterception_test.go

```diff
@@ -83,8 +81,6 @@ func TestInterceptedTxFromFrontendGeneratedParamsGasPriceGasLimitNoData(t *testi
 	)
 }
 
-// testInterceptedTxFromFrontendGeneratedParams tests that a frontend generated tx will pass through an interceptor
-// and ends up in the datapool, concluding the tx is correctly signed and follows our protocol
 func testInterceptedTxFromFrontendGeneratedParams(
```

Comment:
Why did you remove the comment? Please put it back.

Reply:
yes, done